GVRf/Framework/framework/src/main/java/org/gearvrf/GVRCamera.java

```diff
@@ -94,6 +94,7 @@ public void setBackgroundColor(int color) {
      * Meaningful parameter values are from 0 to 1, inclusive: values
      * {@literal < 0} are clamped to 0; values {@literal > 1} are clamped to 1.
      */
+    @Deprecated
```

Comment:
Will deprecate the other one too. Thanks.

I just can't see what good it is to expose setting the clear color per camera. It is not like you will ever need to have different background color in the left eye from the right - that would be one odd thing to look at in VR. I planned to just hide the public APIs, not remove the capability which may be useful to us.

It is much more idiomatic to set the background color on the GVRScene. Easier to discover and code too.